src/nyc_trees/apps/quiz/views.py

```diff
@@ -5,17 +5,11 @@
 
 from django.db import transaction
 
-from apps.quiz.models import Quiz, quizzes
+from apps.quiz.models import Quiz
 from apps.quiz.utils import get_quiz_or_404
 from apps.users.models import TrainingResult
 
 
-def quiz_list_page(request):
-    return {
-        'quizzes': quizzes.iteritems()
-    }
-
-
 def quiz_page(request, quiz_slug):
```

Comment:
I'm pretty sure this route and view are now only used by `/home/training/routes/`, but it is used in a way that no longer makes sense. The training route just calls this with a hardcoded `quiz_slug`, but if we're not going to use a view for arbitrarily named quizzes, shouldn't we flatten that plumming out by removing this view and its route and moving all that code into training?